### ISSUE TITLE
Plugin issue with serverless 3.39.0

### ISSUE THREAD

**Issue by grzes-94** [localstack/serverless-localstack#252]
custom-resources lambda is not being updated properly after updating to serveless 3.39.0 which changed lambda runtime to node18

Plugin searches for `AWS.config.logger = console;` [here](https://github.com/localstack/serverless-localstack/blob/94360032b53bbeb8b4cfe3c2c5a628748410c681/src/index.js#L1011) which was recently removed from [serverless code](https://github.com/serverless/serverless/commit/d40e9f4b323a737a0b13b35f705bfa54d6614a9b#diff-fda9af3217987836f40721ac0d16facce3b0834355e85d35cc7958ff5664cc2eL8).

In a result `AWS.config.s3ForcePathStyle = true;` is added at the end of the utils.js file
```js
module.exports = {
  logger,
  response,
  getEnvironment,
  getLambdaArn,
  handlerWrapper,
  wait,
  MAX_AWS_REQUEST_TRY,
};AWS.config.s3ForcePathStyle = true;
```

Running lambda fails with exception as node18 lambdas uses AWS SDK v3
```
2024-06-18T09:45:55.149Z	undefined	ERROR	Uncaught Exception 	
{
    "errorType": "ReferenceError",
    "errorMessage": "AWS is not defined",
    "stack": [
        "ReferenceError: AWS is not defined",
        "    at Object.<anonymous> (/var/task/utils.js:107:3)",
        "    at Module._compile (node:internal/modules/cjs/loader:1364:14)",
        "    at Module._extensions..js (node:internal/modules/cjs/loader:1422:10)",
        "    at Module.load (node:internal/modules/cjs/loader:1203:32)",
        "    at Module._load (node:internal/modules/cjs/loader:1019:12)",
        "    at Module.require (node:internal/modules/cjs/loader:1231:19)",
        "    at require (node:internal/modules/helpers:177:18)",
        "    at Object.<anonymous> (/var/task/api-gateway-cloud-watch-role/handler.js:3:39)",
        "    at Module._compile (node:internal/modules/cjs/loader:1364:14)",
        "    at Module._extensions..js (node:internal/modules/cjs/loader:1422:10)"
    ]
}
```

**Comment by dfangl** [comment]
Thank you for the detailed report and investigation!
We will work on a patch to make serverless-localstack compatible with the latest serverless version again, and update you here on the progress.

**Comment by dfangl** [comment]
This issue should now be fixed with #254. A new version of the plugin was released minutes ago.
Can you confirm your issues are fixed?

Thank you for reporting this bug!

**Comment by grzes-94** [comment]
Its working as expected now, thanks!
